webui/react/src/hooks/useSettings.ts

```diff
@@ -56,6 +56,7 @@ export interface SettingsConfigProp {
 }
 
 export interface SettingsConfig {
+  applicableRoutespace?: string;
```

Comment:
basically `offset` exists on two settings at the same time, they are erroneously getting each others' settings from the url. 

giving them their own routespace prevents this.